bridge/client/app/_components/ktb-http-loading-bar/ktb-http-loading-bar.component.ts

```diff
@@ -16,70 +16,80 @@ export class KtbHttpLoadingBarComponent implements OnInit, OnDestroy {
   private hideLoadingTimer?: ReturnType<typeof setTimeout>;
   private animateLoadingBarInterval?: ReturnType<typeof setInterval>;
 
-  public loading = false;
+  public loading = 0;
```

Comment:
Could you rename this variable to know what it is used for? Because `loading` sounds more like a `boolean` if it is loading or not.